README.md

```diff
@@ -3,22 +3,42 @@
 [![Board Status](https://dev.azure.com/tc-tibo/4dd797bf-468b-4734-809d-7a5d7e801e2d/98f281cb-b20c-4b34-9384-beeaf67bef78/_apis/work/boardbadge/04d8c3b1-ac45-422d-9c05-c39db02d017d?columnOptions=2&columns=To%20Do,Doing,Done)](https://dev.azure.com/tc-tibo/4dd797bf-468b-4734-809d-7a5d7e801e2d/_boards/board/t/98f281cb-b20c-4b34-9384-beeaf67bef78/Microsoft.RequirementCategory/)
 
 # Introduction 
-TODO: Give a short introduction of your project. Let this section explain the objectives or the motivation behind this project. 
+
+This is the prototype for TC's Seafarer Credentials Project. It is written in Blazor and is strictly for development purposes. Actual development will be hosted in TC's official Github home.
 
 # Getting Started
-TODO: Guide users through getting your code up and running on their own system. In this section you can talk about:
+
 1.	Installation process
+
+To install this project, clone locally, open https://github.com/tc-tibo/SeaCertsBlazor/blob/master/CDTS%20Blazor.sln file in Visual Studio, and then publish to your desired location.
```

Comment:
To install this project, clone the project locally and open the .sln file to access the solution in Visual Studio.

Reply:
Shouldn't we have a project that is not IDE dependent? readme shouldn't really presume a particular IDE

Reply:
Then maybe just having the instructions to clone the repo and that's it?

Reply:
> > I received: "no build task to run found. Configure Build Task" when I tried to build.
> > We should maybe include how to do that?
> 
> I'm assuming this is because you're using VS code. The solution works out of the box if you use Visual Studio instead of Visual Studio code. That being said, it could be a good idea to include a how-to guide for Visual Studio code or add it as a task that someone can accomplish at a future date. I'm not certain we, as a core team, should embark on putting a guide for every IDE.
> 
> I've created an issue though so anyone can handle it within our team or outside #10.

Shouldn't we have a project that is not IDE dependent? readme shouldn't really presume a particular IDE